scripts/ci/ci.sh

```diff
@@ -11,7 +11,7 @@ if [[ ("$OS_NAME" == "linux") && ("$BUILDKITE_BRANCH" == "master") ]];
   else with_haddock=false
 fi
 
-targets="cardano-sl cardano-sl-auxx cardano-sl-tools cardano-sl-wallet"
+targets="cardano-sl cardano-sl-auxx cardano-sl-tools cardano-sl-wallet cardano-sl-wallet-new"
```

Comment:
we should delete `cardano-sl-wallet`

Reply:
@domenkozar If this is building the executable yes, but `cardano-wallet-new` has still a _library_ dependency on `cardano-wallet`, so maybe that's fine it keep it there for now 😉 (but you tell me, ofc).